docs/components/sdk_support_table.js

```diff
@@ -24,7 +23,7 @@ export default class SDKSupportTable extends React.Component {
                         </tr>
                     </thead>
                     <tbody>
-                        {entries(this.props).map(([key, entry], i) => <tr key={i}>
+                        {Object.entries(this.props).map(([key, entry], i) => <tr key={i}>
```

Comment:
`Object.entries` is not supported in IE. Do we have a polyfill somewhere for it?

Reply:
Oh, I didn't realize this is executing at runtime rather than compiling.

Reply:
Ah I could be wrong. I just know that we had an issue in the current release where docs weren't working in IE11 because of an Object.assign call so I assumed this could be an issue as well.